src/Entities/Player.cpp

```diff
@@ -146,6 +146,22 @@ cPlayer::cPlayer(cClientHandlePtr a_Client, const AString & a_PlayerName) :
 
 
 
+void cPlayer::InitializePlayer()
```

Comment:
Doesn't matter, just don't use the parameter.


On 15 Feb 2016, at 11:55, "LogicParrot" <notifications@github.com> wrote:

> In src/Entities/Player.cpp:
> 
> > @@ -146,6 +146,22 @@ cPlayer::cPlayer(cClientHandlePtr a_Client, const AString & a_PlayerName) :
> >  
> >  
> >  
> > +void cPlayer::InitializePlayer()
> @tigerw Now I see why I haven't done this before. It's because the player version does not need a world parameter.
> 
> —
> Reply to this email directly or view it on GitHub.
>